math_.py

```diff
@@ -13,3 +13,14 @@ def substract(a: int, b: int) -> int:
 def list_substract(a: List, b: List) -> List:
     assert len(a)==len(b), "len(a) != len(b)"
     return [i-j for i, j in zip(a,b)]
+
```

Comment:
This function looks like it's trying to determine if a value is a number. The name `isnum` isn't very descriptive. Consider renaming it to something like `is_numeric` or `is_integer` based on the intended behavior.